src/battle/actor/goomba.c

```diff
@@ -0,0 +1,443 @@
+#include "common.h"
+#include "battle/battle.h"
+#include "script_api/battle.h"
+#include "goomba.h"
+
+ApiStatus func_8021818C_430B2C(ScriptInstance* script, s32 isInitialCall);
+s32 idleAnimations_80219760[];
+s32 idleAnimations_80219714[];
+s32 defenseTable_80219610[];
+s32 statusTable_8021961C[];
+s32 defenseTable_80219610[];
+s32 partsTable_802196C8[];
+Script script_Init_802197AC;
+Script script_TakeTurn_8021A300;
+Script script_Idle_802197F8;
+Script script_HandleEvent_80219AD4;
+
+// 431FB0-431FBC (VRAM: 80219610)
+s32 defenseTable_80219610[] = {
+    0x00000001, 0x00000000, 0x00000000,
+};
+
+// 431FBC-432068 (VRAM: 8021961C)
+s32 statusTable_8021961C[] = {
+    0x00000001, 0x00000000, 0x00000002, 0x00000000, 0x00000006, 0x00000064, 0x00000009, 0x00000064,
+    0x00000007, 0x00000064, 0x00000004, 0x00000064, 0x00000003, 0x00000064, 0x0000000B, 0x00000064,
+    0x00000005, 0x00000064, 0x0000000A, 0x00000064, 0x00000008, 0x00000064, 0x0000001F, 0x00000000,
+    0x00000020, 0x00000000, 0x00000025, 0x00000000, 0x00000022, 0x00000000, 0x00000024, 0x00000000,
+    0x00000023, 0x00000000, 0x00000021, 0x00000000, 0x00000026, 0x00000000, 0x00000027, 0x00000000,
+    0x00000029, 0x00000000, 0x00000000,
+};
+
+// 432068-43208C (VRAM: 802196C8)
```

Comment:
@ethteck @pixel-stuck do we want to include these after data is matched or should I delete them

Reply:
They should be deleted imo.

Reply:
yeah I think we're good. We can use the map file / tools if we need to locate this stuff